odoo/addons/test_impex/tests/test_load.py

```diff
@@ -976,7 +1005,6 @@ def test_multi_fullsplit(self):
         self.assertEqual(values(b.child1), [11, 12, 13, 14])
         self.assertEqual(values(b.child2), [21, 22, 23])
 
-
```

Comment:
why?

Reply:
No reason so probably an errorneous kill-line I missed when committing. I'll revert it.